ruby/tests/well_known_types_test.rb

```diff
@@ -165,5 +168,17 @@ def test_struct_nested_assign
     assert_equal expected_b_x, s['b']['x'].values
     assert_equal true, s['b']['y']
```

Comment:
I would like to see these access checks made with string, symbols, and a combination, to make it really clear they are checking the same thing.

```ruby
assert_equal true, s['b']['y']
assert_equal true, s[:b][:y]
assert_equal true, s[:b]['y']
assert_equal true, s['b'][:y]
```

Reply:
Makes sense - added.